lib/matplotlib/font_manager.py

```diff
@@ -1419,8 +1418,13 @@ def findfont(prop, fontext='ttf'):
 
     def _rebuild():
         global fontManager
+
         fontManager = FontManager()
+
         if _fmcache:
+            with cbook.Locked(_fmcache):
```

Comment:
should use just `cachedir` here.  It looks like this is creating a folder with the relevant name.